src/qe/lite/qel.cpp

```diff
@@ -0,0 +1,54 @@
+/*++
+
+  Module Name:
+
+    qel.cpp
+
+Abstract:
+
+    Light weight partial quantifier-elimination for cubes using term graphs
```

Comment:
Remove partial? Also, we have boolean projection so some non-cubes are supported?
In any case, best effort for non-cubes should also work (not crash), but we may end up doing nothing.